src/parse-stream.js

```diff
@@ -535,6 +543,11 @@ export default class ParseStream extends Stream {
         ['BYTERANGE-START', 'BYTERANGE-LENGTH'].forEach(function(key) {
           if (event.attributes.hasOwnProperty(key)) {
             event.attributes[key] = parseInt(event.attributes[key], 10);
+
+            const subkey = key === 'BYTERANGE-LENGTH' ? 'length' : 'offset';
```

Comment:
This one is a bit different as we have attributes for both `offest` and `length`